docs/CHANGELOG.md

```diff
@@ -0,0 +1,367 @@
+<!--template:
+# unreleased - YYYY-MM-DD
+## Python API
+
+## CLI
+
+## Catalog
+!-->
+
+# unreleased - YYYY-MM-DD
+## Python API
+* [performance] Improve performance of hashing of S3 files
+  ([#1816](https://github.com/quiltdata/quilt/issues/1816))
+
+## CLI
+* [feature] Add `meta` argument to the push command
+  ([#1793](https://github.com/quiltdata/quilt/issues/1793))
+
+## Catalog
```

Comment:
@nl0 any suggestions on the format?

Reply:
I don't know how often the catalog is deployed. And is it deployed on its own or alongside with labdas.

I can suggest only:

## [2020-12-31 12:48]
* [feature] New shiny feature
* [fix] Fix ugly bug

Time of the day is relevant if there are multiple deployment throughout the day.

Reply:
since there's no separate release process for the catalog, we should keep all the changes under one version, as it were before, and the current format (`x.y.z - YYYY-MM-DD`) works fine for this imo.

we should agree on prefixes (types of changes) tho, like, i dont think `feature` is a good prefix in this context, i'd rather start with the types suggested by [keep a changelog](https://keepachangelog.com/en/1.0.0/#how), and maybe added `Performance` -- it makes perfect sense (maybe some other types will come up later).